scripts/jenkins/qa_openstack.sh

```diff
@@ -120,6 +120,9 @@ case "$cloudsource" in
     ;;
 esac
 
+# be sure to use packages from cloud repository
+$zypper dup --no-recommends --from cloud
```

Comment:
Then I don't have the cloud-init magic when I reboot the instance. It's not really conflicting with the base operating system, it conflicts with the public cloud module which is used to build the image.